pallets/df-integration-tests/src/tests/faucets.rs

```diff
@@ -15,7 +15,7 @@ impl ExtBuilder {
         let mut ext = Self::build();
         ext.execute_with(|| {
             assert_ok!(_add_default_faucet());
-            assert_ok!(FaucetsMembership::add_member(Origin::signed(SUDO_ACCOUNT), FAUCET1));
+            assert_ok!(OffchainMembership::add_member(Origin::signed(SUDO_ACCOUNT), FAUCET1));
```

Comment:
It's better to test it with another account that is not sudo.
Maybe something like `ACCOUNT_OFFCHAIN = 123`